advanced/fix_files.sh

```diff
@@ -0,0 +1,15 @@
+# Touch files to ensure they exist (create if non-existing, preserve if existing)
```

Comment:
Can we have a shebang on this? Probably should throw in the Pi-hole copyright header on it as well.
```suggestion
#!/usr/bin/env bash
# Pi-hole: A black hole for Internet advertisements
# (c) 2017 Pi-hole, LLC (https://pi-hole.net)
# Network-wide ad blocking via your own hardware.
#
# Completely uninstalls Pi-hole
#
# This file is copyright under the latest version of the EUPL.
# Please see LICENSE file for your rights under this license.
```

Reply:
I've added the shebang and copyright header, although it has changed from the original where it sets the owner to `$USER` instead of `pihole`. For the modified `init.d/pihole-FTL` and the systemd definition it is correct but directly calling the file might produce unexpected behavior.